### PR TITLE
[12.0][MIG]stock_analytic

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Work in progress as some refactoring in this module is probably needed.

### CODE REVIEW

File: stock_analytic/views/stock_move_views.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (Nikul-Chaudhary) [inline stock_analytic/views/stock_move_views.xml:3]: @aheficent Remove, No need

File: stock_analytic/views/stock_move_views.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <record id="view_move_picking_form" model="ir.ui.view">
+            <field name="name">stock.move.picking.form</field>
+            <field name="model">stock.move</field>
+            <field name="inherit_id" ref="stock.view_move_picking_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='location_dest_id']" position="after">
```
Comment (Nikul-Chaudhary) [inline stock_analytic/views/stock_move_views.xml:9]: @aheficent IMO Use direct Field here, Or other `xpath`

File: stock_analytic/tests/test_stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        self.product = self.env.ref('product.product_product_4')
+        self.product_categ = self.env.ref('product.product_category_5')
+        self.valuation_account = self.env['account.account'].create({
+            'name': 'Test stock valuation',
+            'code': 'tv',
+            'user_type_id':
+                self.env['account.account.type'].search(
+                    [], limit=1).id,
+            'reconcile': True,
+            'company_id': self.env.ref('base.main_company').id,
```
Comment (Nikul-Chaudhary) [inline stock_analytic/tests/test_stock_picking.py:26]: @aheficent IMO Use this `self.env.company.id` company_id, same as others

File: stock_analytic/tests/test_stock_picking.py [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        }
+
+        picking = self.env['stock.picking'].create(picking_data)
+
+        move_data = {
+            'picking_id': picking.id,
+            'product_id': self.product.id,
+            'location_id': location_id.id,
+            'location_dest_id': location_dest_id.id,
+            'date': datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
```
Comment (Nikul-Chaudhary) [inline stock_analytic/tests/test_stock_picking.py:83]: @aheficent here use `server` time

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improved Code

**Comment by AaronHForgeFlow** [comment]
@Nikul-Chaudhary comments will be attended on #223 Sorry for the mistake.

**Comment by Nikul-Chaudhary** [comment]
> @Nikul-Chaudhary comments will be attended on #223 Sorry for the mistake.

Noted!!

**Pull request closed without merging**